static/app/views/performance/landing/widgets/widgetDefinitions.tsx

```diff
@@ -267,14 +268,16 @@ export const WIDGET_DEFINITIONS: ({
   [PerformanceWidgetSetting.MOST_SLOW_FRAMES]: {
     title: t('Most Slow Frames'),
     titleTooltip: getTermHelp(organization, PERFORMANCE_TERM.SLOW_FRAMES),
-    fields: ['p75(measurements.frames_slow_rate)'],
+    fields: ['avg(measurements.frames_slow)', 'avg(measurements.frames_slow_rate)'],
+    sortField: 'avg(measurements.frames_slow)',
     dataType: GenericPerformanceWidgetDataType.line_list,
     chartColor: WIDGET_PALETTE[0],
   },
   [PerformanceWidgetSetting.MOST_FROZEN_FRAMES]: {
     title: t('Most Frozen Frames'),
     titleTooltip: getTermHelp(organization, PERFORMANCE_TERM.FROZEN_FRAMES),
-    fields: ['p75(measurements.frames_frozen_rate)'],
+    fields: ['avg(measurements.frames_frozen)', 'avg(measurements.frames_frozen_rate)'],
```

Comment:
A `LineChartListWidget` ignores the second field, and why is this using `avg` now?

Reply:
Slow/Frozen frames are somewhat intermittent and usually are a very low spread, when every value is 1 or 2 (for frozen frames, for example) all the lines look the same, it's hard to know whether a transaction is truly the cause of a bunch of frame drops. Essentially I think using quantile for such a small population gap might not be the best.

Reply:
Also, it shouldn't be ignoring the second field, so I'll fix that, we likely want to maintain sort across rate since it normalizes for low throughput transactions.